content/en/docs/installing-on-linux/introduction/storage-configuration.md

```diff
@@ -0,0 +1,119 @@
+# Overview
```

Comment:
Please add the header for this file, example:

```
---
title: "Persistent Storage Configuration"
keywords: 'kubernetes, docker, kubesphere, storage, volume, PVC'
description: 'Persistent Storage Configuration'

linkTitle: "Persistent Storage Configuration"
weight: 2110
---
```

Reply:
done